### PR TITLE
Modification pour integration du module Customize Prospect Customer F…

### PR DESCRIPTION
**Pull request by thomasjulie**
Modification pour integration du module Customize Prospect Customer Field. Ref Dolistore : c21041d20230810101735

Ce module permet la personnalisation de la liste déroulante "Prospect / Client" du le module "Tiers".
Ajout et utilisation d'une nouvelle fonction "selectProspectCustomerTypeCustom"

This module allow the customization of the dropdown list "Prospect / Customer" in the module "Third Parties".
Adding and using a new function "selectProspectCustomerTypeCustom"

### CODE REVIEW

**Comment by hregis** [comment]
@thomasjulie why change "label" to "libelle" ?

**Comment by thomasjulie** [comment]
We used the wrong dolibarr version (17 instead of develop).
Sorry for this, we edited our pull request using the version of the develop branch.

File: htdocs/core/class/html.formcompany.class.php [hunk 1]
```diff
@@ -1032,47 +1032,90 @@ public function selectProspectCustomerType($selected, $htmlname = 'client', $htm
...
 
-		$out = '<select class="flat ' . $morecss . '" name="' . $htmlname . '" id="' . $htmlidname . '">';
-		if ($typeinput == 'form') {
-			if ($allowempty || ($selected == '' || $selected == '-1')) {
-				$out .= '<option value="-1">';
-				if (is_numeric($allowempty)) {
-					$out .= '&nbsp;';
-				} else {
-					$out .= $langs->trans($allowempty);
+		if (!$conf->customizeprospectcustomerfield->enabled) {
```
Comment (frederic34) [inline htdocs/core/class/html.formcompany.class.php:1035]: @thomasjulie add a hook to be able to personalize

Comment (eldy) [reply htdocs/core/class/html.formcompany.class.php:1035]: You can take a look at this page to know how to
https://wiki.dolibarr.org/index.php/Hooks_system#Adding_a_hook_point_into_code

**Pull request closed without merging**